qa/suites/upgrade/hammer-jewel-x/stress-split/1-hammer-install-and-upgrade-to-jewel/hammer-to-jewel.yaml

```diff
@@ -63,6 +61,7 @@ upgrade-sequence-h-j:
   - exec:
       mon.a:
       - sleep 300
+      - ceph osd set sortbitwise
```

Comment:
can we reuse `qa/releases/jewel.yaml` instead, i.e.  add an extra step, like `1.5-jewel.yaml`, and link it to this file?